web/src/features/fba/components/map/Legend.tsx

```diff
@@ -41,46 +41,50 @@ interface LegendItemProps {
   description?: string | null
 }
 
-const LegendItem: React.FC<LegendItemProps> = ({ label, checked, onChange, subItems, description }) => (
-  <div>
-    <Grid>
-      <Grid container alignItems={'center'}>
-        <Grid item>
-          <Checkbox
-            data-testid={`${label.toLowerCase().split(' ')[0]}-checkbox`}
-            checked={checked}
-            onChange={onChange}
-          />
-        </Grid>
-        <Grid item>
-          <Typography variant="h2" sx={{ fontSize: '1rem', fontWeight: 'bold' }}>
-            {label}
-          </Typography>
+const LegendItem: React.FC<LegendItemProps> = ({ label, checked, onChange, subItems, description }) => {
+  // Dirty hack used to force whitespace for the Snow Coverage legend item
+  const hasEmptyDescription = description === ''
+  return (
+    <div>
+      <Grid>
+        <Grid container alignItems={'center'}>
+          <Grid item>
+            <Checkbox
+              data-testid={`${label.toLowerCase().split(' ')[0]}-checkbox`}
+              checked={checked}
+              onChange={onChange}
+            />
+          </Grid>
+          <Grid item>
+            <Typography variant="h2" sx={{ fontSize: '1rem', fontWeight: 'bold' }}>
+              {label}
+            </Typography>
+          </Grid>
         </Grid>
-      </Grid>
-      <Grid container alignItems={'center'}>
-        <Grid item sx={{ transform: 'translate(50%, -50%)' }}>
-          <Typography variant="body1" sx={{ fontSize: '0.75rem' }}>
-            {description}
-          </Typography>
+        <Grid container alignItems={'center'}>
+          <Grid item sx={{ transform: 'translate(50%, -50%)' }}>
+            <Typography variant="body1" sx={{ fontSize: '0.75rem' }}>
+              {hasEmptyDescription ? <span>&nbsp;</span> : description}
```

Comment:
I wouldn't have initially thought of the `&nbsp`, but I think after seeing it the whole thing could maybe be done with:
`description ?? <span>&nbsp;</span>`

Might be wrong about that, looks great regardless!

Reply:
Good thought, but we only want this blank space below the 'Snow Coverage' item. Using `description ?? <span>&nbsp;</span>` would result in a blank space below the other two legend items as well.

Your comment did give me an idea for making this slightly less hacky though. Thanks!

Reply:
Ah of course